index.d.ts

```diff
@@ -1,8 +1,18 @@
+interface Options {
+    /**
```

Comment:
Use tab-indentation.

Reply:
Fixed, the 10 other projects i'm involved in do not use tabs, sorry, its hard to put in tabs when the editor replaces them with spaces.

Reply:
Hi @dl748, since this project uses an `.editorconfig` file, your editor should be able to hand this "tab vs spaces" thing automatically for you. If you're using VSCode, try globallin installing the `editorconfig` package (`npm i -g editorconfig`) and enabling the `EditorConfig` extension in VSCode. It should work automagically.

Reply:
Thanks but I don't use VSCode.